bciers/apps/compliance/src/tests/components/compliance-summaries/ComplianceSummariesPage.test.tsx

```diff
@@ -82,4 +93,38 @@ describe("ComplianceSummariesPage", () => {
     ).toBeVisible();
     expect(screen.getByText(/due on November 30, 2025/i)).toBeVisible();
   });
+
+  it("renders alert message with 'was' if current date is past due", async () => {
+    const currentDate = new Date("2025-12-01");
+    vi.setSystemTime(currentDate);
+    await renderPage();
+    expect(
+      screen.getByText(
+        /Your compliance obligation for the 2024 reporting year was/i,
+      ),
+    ).toBeVisible();
+    expect(screen.getByText(/due on November 30, 2025/i)).toBeVisible();
+  });
+
+  it("doesn't render alert message for overdue penalty if date is before due", async () => {
+    const currentDate = new Date("2025-10-01");
```

Comment:
Suggest additional tests for a second before midnight on 2025-11-30 and the stroke of midnight 2025-12-01 to make sure the alerts display as expected